firebase-firestore/src/main/java/com/google/firebase/firestore/remote/Datastore.java

```diff
@@ -63,9 +63,9 @@
    * part of their app's dependencies.
    */
   static final String SSL_DEPENDENCY_ERROR_MESSAGE =
-      "The Firestore SDK failed to establish a secure connection. This is likely a problem with "
-          + "your app, rather than with Firestore itself. See https://bit.ly/2XFpdma for "
-          + "instructions on how to enable TLS on Android 4.x devices.";
+      "The Cloud Firestore SDK failed to establish a secure connection. This is likely a problem "
```

Comment:
This is wrong but the correct text is probably too wordy. This isn't a "Cloud Firestore SDK", this is the "Cloud Firestore for Firebase SDK" :-(. In the other error messages we refer to things like this "Cloud Firestore client's offline persistence". Maybe s/SDK/client/?